js/app/src/components/Video/Video.svelte

```diff
@@ -57,8 +57,6 @@
 		} else {
 			value = null;
 		}
-
-		dispatch("change");
```

Comment:
Not needed because we already dispatch change if the value changes in the subsequent lines